### PR TITLE
Update FAQ, add permission check for Q4

### PR DESCRIPTION
**Pull request by fengou1**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

1. Numbering the question, so that people easy to refer to question, like 'Q14 will resolve your issue'
2. As a new comer, I have met the Q4 again, however Question does not provide the way to check permission/user, Add those info so that people could follow it to resolve the issue.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@fengou1 I am afraid this PR cannot be merged if the user "phone" does not sign the Contributor License Agreement as indicated in https://github.com/pingcap/docs-cn/pull/6497#issuecomment-866501283.

File: br/backup-and-restore-faq.md [hunk 1]
```diff
@@ -44,57 +44,75 @@ aliases: ['/docs-cn/dev/br/backup-and-restore-faq/']
...
 
 会的，BR 会备份表的 [`SHARD_ROW_ID_BITS` 和 `PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) 信息，并恢复成多个 Region。
 
-## 使用 BR 恢复备份数据后，SQL 查询报错 `region is unavailable`
+## Q11: 使用 BR 恢复备份数据后，SQL 查询报错 `region is unavailable`
 
 如果 BR 备份的集群有 TiFlash，恢复时会将 TiFlash 信息存进 `TableInfo`。此时如果恢复的集群没有 TiFlash，则会报该错误。计划在未来版本中修复该错误。
 
-## BR 是否支持就地 (in-place) 全量恢复某个历史备份？
+## Q12: BR 是否支持就地 (in-place) 全量恢复某个历史备份？
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:103]: I don't think it is good for doc maintenance. When one FAQ is removed, all FAQ titles must be updated, which might lead to problems.

Comment (fengou1) [reply br/backup-and-restore-faq.md]: Got it, Remove number.

**Comment by fengou1** [comment]
PR involved a non-github account, it can not merged, so close this PR, create a new one late.

**Pull request closed without merging**